router/clientRouter.js

```diff
@@ -24,8 +25,12 @@ router
 
 // Images Routes
   .post('/image', images.addImage)
-  .get('recent', suggestions.getRecent)
-  .get('suggestions', suggestions.getSuggestions);
+  .get('/api/recent', suggestions.getRecent)
```

Comment:
Do you intend for these routes to be /api/api/recent? The client router is used inside of the /api URI